### DISCUSSION TITLE
Old Bugzilla-Issues

### DISCUSSION THREAD

**Discussion by eugen-eugen** [eclipse-cdt/cdt#300] [Q&A]
Hallo, everyone,
where can old CDT-Issues, which were in eclipse bugzilla 15-10 years ago be accessed yet?
Thanks and BR!

**Comment by jonahgraham** [comment]
You can find them still in Bugzilla here: https://bugs.eclipse.org/bugs/buglist.cgi?product=CDT but this [table](https://bugs.eclipse.org/bugs/report.cgi?bug_file_loc=&bug_file_loc_type=allwordssubstr&bug_id=&bug_id_type=anyexact&chfieldfrom=&chfieldto=Now&chfieldvalue=&email1=&email2=&email3=&emailtype1=substring&emailtype2=substring&emailtype3=substring&field0-0-0=noop&keywords=&keywords_type=allwords&longdesc=&longdesc_type=allwordssubstr&short_desc=&short_desc_type=allwordssubstr&status_whiteboard=&status_whiteboard_type=allwordssubstr&type0-0-0=noop&value0-0-0=&votes=&votes_type=greaterthaneq&x_axis_field=resolution&y_axis_field=bug_status&z_axis_field=&width=600&height=350&action=wrap&format=table&product=CDT) or [this one](https://bugs.eclipse.org/bugs/report.cgi?x_axis_field=resolution&y_axis_field=component&z_axis_field=&no_redirect=1&query_format=report-table&short_desc_type=allwordssubstr&short_desc=&product=CDT&longdesc_type=allwordssubstr&longdesc=&bug_file_loc_type=allwordssubstr&bug_file_loc=&status_whiteboard_type=allwordssubstr&status_whiteboard=&keywords_type=allwords&keywords=&deadlinefrom=&deadlineto=&bug_id=&bug_id_type=anyexact&votes=&votes_type=greaterthaneq&emailtype1=substring&email1=&emailtype2=substring&email2=&emailtype3=substring&email3=&chfieldvalue=&chfieldfrom=&chfieldto=Now&j_top=AND&f1=noop&o1=noop&v1=&format=table&action=wrap) but be more navigable

Also we include links to bugzillas fixed in releases in N&N like here https://github.com/eclipse-cdt/cdt/blob/main/NewAndNoteworthy/CDT-11.0.md#bugs-fixed-in-this-release
